Index.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+   <main id="main-doc">
+    <nav id="navbar">
+        <header>Technical Documentation of How to Drive for Newbies</header>
+    <ul>
+        <li><a class="nav-link" href="#Introduction">Introduction</a></li>
+        <li><a class="nav-link" href="#Instructions">Instructions</a></li>
+        <li><a class="nav-link" href="#Operating">Operating</a></li>
+        <li><a class="nav-link" href="#Driving">Driving</a></li>
+        <li><a class="nav-link" href="#Overview">Overview</a></li>
+    </ul>
+    </nav>
+<Section class = "main-section" id="Introduction"><header>Introduction</header>
+<p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Id, odit. Cum doloribus pariatur nam alias aspernatur repellendus architecto aliquid natus sequi ab voluptates amet magni incidunt ullam, quam error perferendis.</p>
+<p>how is the studies going</p>
+<p>hi Muzz</p>
+<code><?php echo 'hi';></code>
+<ul> 
+    <li>Lorem</li>
+    <li>Lorem</li>
+    <li>Lorem</li>
+    <li>Lorem</li>
+</ul>
+<code>hj</code>
+<h1>Next</h1>
```

Comment:
I think that you should use a different h tag other than h1 because h1 is for the main header and here you are now at the closing of the section